client/app/bundles/shared/components/Logo.jsx

```diff
@@ -7,14 +7,15 @@ type Props = {
   size?: string; // Future Task: Use ScreenSize enum
 }
 
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
I see this is disabled multiple times, should we just disable it globally instead of by a case-by-case instance?

Reply:
Yeah let's turn it off, I think it would matter more if all components are stateless redux ones. Atm it's not helping

Reply:
Cool beans!